tests/unit/conftest.py

```diff
@@ -83,4 +83,9 @@ def __init__(self):
         def config(self):
             return self.data
 
+        def is_ready(self, obj, cond):
+            if obj.kind == "CustomResourceDefinition" and cond.type == "Ignored":
+                return None
+            return cond.status == "True"
```

Comment:
example of how to override the Manifest object to ignore certain conditions on objects